### PR TITLE
feat(IT Wallet): [SIW-1471] Add skeumorphic card for the EU Disability Card

### PR DESCRIPTION
**Pull request by mastro993**
## Short description
This PR adds the skeumorphic card for the EU Disability Card 

## List of changes proposed in this pull request
- Added skeumorphic card assets
- Added data rendering
- Fixed incorrect claims order in MDL front card
- Extended `DebugPrettyPrint` component with the ability to share its content via device's sharing capabilities

## How to test
This PR needs to be tested on different screen sizes. 
Navigate to the DC credential details screen, check that the card is displayed correctly.
Alternatively, the component can be tested under **Options > Design System > Cards > Documents**. Scroll down to see the Skumorphic card of the EUDC.

## Preview

https://github.com/user-attachments/assets/2e8fd70d-47dd-4599-9fbe-a05e1540590e

### CODE REVIEW

File: ts/features/itwallet/common/components/ItwSkeumorphicCard/CardClaim.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ export type ClaimDimensions = Prettify<
 export type CardClaimProps = Prettify<
   {
     // A claim that will be used to render its component
-    claim: ParsedCredential[number];
+    claim?: ParsedCredential[number];
```
Comment (LazyAfternoons) [inline ts/features/itwallet/common/components/ItwSkeumorphicCard/CardClaim.tsx:44]: Question: why is the claim optional? Can we add a comment about it?

Comment (mastro993) [reply ts/features/itwallet/common/components/ItwSkeumorphicCard/CardClaim.tsx:45]: Addressed here https://github.com/pagopa/io-app/pull/6184/commits/929f4f25e89016b505730db06cc7795412acc9d8

**Review by dmnplb — APPROVED** [review]

**Review by LazyAfternoons — COMMENTED** [review]
The mock credential doesn't seem to be updated but that's marginal. 
Is the name supposed to be uppercase?

**Review by LazyAfternoons — COMMENTED** [review]
There's an issue with the QR Code which doesn't seem to be centered on my devices: 
| Before | After |
|--------|--------|
|  |  |

**Comment by mastro993** [comment]
> There's an issue with the QR Code which doesn't seem to be centered on my devices:
> 
> 5.7"	5.15"
> 	

Here https://github.com/pagopa/io-app/pull/6184/commits/26a385e9d7353fc9db903089f7405dc629e249c4, I've improved the positioning and sizing for QR Code images using relative values. Now it should display the QR Code in the correct size and position.

**Review by LazyAfternoons — APPROVED** [review]
LGTM

**Pull request merged**